modules/va_facilities/README.v1.yml

```diff
@@ -449,6 +449,11 @@ paths:
         in the standard JSON API "links" and "meta" elements. GeoJSON responses include pagination
         information in the "Link" header.
 
+        Also in the "meta" element is information about the drive time bands returned for the facilities
+        in the response. The values of drive_time_band_min and drive_time_band_max are in minutes.  
```

Comment:
I'm trying to make this language more clear, but realize this I might be going overboard. Does this make sense?
> Also in the "meta" element is information about the drive time band that the requested location fell within for each the facility included in the response.

Reply:
Yes, thank you!